src/ConjugatePriors.jl

```diff
@@ -2,12 +2,19 @@ module ConjugatePriors
 
 using Statistics
 using LinearAlgebra
+has_cholesky(a::AbstractMatrix) = isa(try_cholesky(a), Cholesky)
+function try_cholesky(a::AbstractMatrix)
+    try cholesky(a)
+    catch e
+        return e
+   end
+end
```

Comment:
Couldn't we just use `LinearAlgebra.isposdef` (or possibly even `LinearAlgebra.isposdef!`)?
```suggestion
```